crates/parcel_core/src/plugin/transformer_plugin.rs

```diff
@@ -26,103 +25,70 @@ pub struct ResolveOptions {
 /// A function that enables transformers to resolve a dependency specifier
 pub type Resolve = dyn Fn(PathBuf, String, ResolveOptions) -> Result<PathBuf, anyhow::Error>;
 
-/// Compile a single asset, discover dependencies, or convert the asset to a different format
```

Comment:
/nit The first line of docs should summarise the intent and only be on a single line if it later appears in docs